packages/target-tauri/src-tauri/tauri.conf.json5

```diff
@@ -62,7 +62,9 @@
             // open current logfile & open links
             "shell:allow-open",
             // Badge counter
-            "core:window:allow-set-badge-count"
+            "core:window:allow-set-badge-count",
+            "core:menu:default",
+            "core:menu:allow-new"
```

Comment:
please add comment for what this is needed

Reply:
is this even needed? as far as I understand it these capabilities limit only what you are able to call from javascript.